plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -130,6 +167,13 @@ private static void installElasticsearchPlugin(
                 .put("elasticsearch.scroll-size", "1000")
                 .put("elasticsearch.scroll-timeout", "1m")
                 .put("elasticsearch.request-timeout", "2m")
+                .put("elasticsearch.tls.enabled", "true")
+                .put("elasticsearch.tls.truststore-path", new File(getResource("truststore.jks").toURI()).getPath())
+                .put("elasticsearch.tls.truststore-password", "123456")
```

Comment:
`PASSWORD`?

Reply:
We usually use this one for statically generated JKS